### PR TITLE
Add support for Python 3.12

### PR DESCRIPTION
**Pull request by vdusek**
Closes https://github.com/apify/apify-core/issues/12587

### CODE REVIEW

**Comment by vdusek** [comment]
In the case of Playwright, blocked until [v1.39.0](https://github.com/microsoft/playwright-python/releases/tag/v1.39.0) is not released on PyPI.

**Comment by vdusek** [comment]
Blocked by https://github.com/apify/apify-actor-docker/issues/113

File: .github/workflows/release-python-playwright.yaml [hunk 1]
```diff
@@ -21,8 +21,8 @@ on:
 env:
   RELEASE_TAG: ${{ github.event.inputs.release_tag || github.event.client_payload.release_tag }}
   APIFY_VERSION: ${{ github.event.inputs.apify_version || github.event.client_payload.apify_version }}
-  PLAYWRIGHT_VERSION: ${{ github.event.inputs.playwright_version || github.event.client_payload.playwright_version || '1.33.0' }}
-  LATEST_PYTHON: "3.11"
+  PLAYWRIGHT_VERSION: ${{ github.event.inputs.playwright_version || github.event.client_payload.playwright_version || '1.39.0' }}
+  LATEST_PYTHON: "3.12"
```
Comment (fnesveda) [inline .github/workflows/release-python-playwright.yaml:25]: Let's keep 3.11 as `LATEST_PYTHON` for a while, I bet there are still a lot of libraries broken on 3.12.

Comment (fnesveda) [reply .github/workflows/release-python-playwright.yaml:25]: `LATEST_PYTHON` is bad naming here, it should be called `DEFAULT_PYTHON`, as it denotes the Python version to which the `apify/actor-python` image without any tag will point to. It's called "latest" as it's the `latest` tag of the image.

**Review by fnesveda — CHANGES_REQUESTED** [review]
Cool! Just one note for the `LATEST_PYTHON` constant

**Review by fnesveda — APPROVED** [review]

**Pull request merged**